dumbo/src/tcp/connection.rs

```diff
@@ -466,9 +465,12 @@ impl Connection {
         }
 
         if v + current_w > max_w {
-            self.local_rwnd_edge = self.ack_to_send + max_w;
+            let adv = self.ack_to_send + max_w;
```

Comment:
Why do you need an auxiliary value?
Why not add a `return` outside of this `if` with `return &self.local_rwnd_edge.0`?

Reply:
The intent was to return the value that the receive window was increased by, not the receive window itself. Would you like me to instead return the window?

Reply:
You are right, the intent of the function is to return the advance value. I was under the false impression.
 However, if that is the case the function still returns an incorrect value for the first branch of the `if`. In the case where `self.local_rwnd_edge` is updated to the max value (i.e `self.ack_to_send + max_w`), the advance value is actually computed by substracting what was previously stored in `self.local_rwnd_edge` and the new value.